src/sonic-yang-mgmt/yang-models/sonic-interface.yang

```diff
@@ -0,0 +1,79 @@
+module sonic-interface {
+
+	namespace "http://github.com/Azure/sonic-interface";
+	prefix intf;
+
+	import ietf-yang-types {
+		prefix yang;
+	}
+
+	import ietf-inet-types {
+		prefix inet;
+	}
+
+	import sonic-head {
+		prefix head;
+		revision-date 2019-07-01;
+	}
+
+	import sonic-port {
+		prefix port;
+		revision-date 2019-07-01;
+	}
+
+	organization "Linkedin Corporation";
+
+	contact "lnos_coders@linkedin.com";
+
+	description "INTERFACE yang Module for SONiC OS";
+
+	revision 2019-07-01 {
+		description "First Revision";
+	}
+
+	container sonic-interface {
+		container INTERFACE {
+
+			description "INTERFACE part of config_db.json";
+
+			list INTERFACE_LIST {
+
+				key "interface ip-prefix";
```

Comment:
Two versions of INTERFACE_LIST.

Reply:
Two version shd not be much different, I will merge in my pr.

Reply:
In our version, we have less leafs. BCRM has complex must condition with other leafs which we do not want to bring without testing.